docs/layers/software-delivery/eks-argocd/tutorials/github-apps.mdx

```diff
@@ -144,13 +132,28 @@ To address security concerns, the following compensating controls should be impl
 
     <Steps>
       1. For each GitHub App:
-        <Steps>
-         1. On the GitHub App page, scroll down to "Private keys" and click "Generate a private key"
-         1. Download the private key file
-         1. Store the App ID, Installation ID, and private key securely in 1Password
-        </Steps>
-      2. Upload these credentials to AWS SSM Parameter Store:
-         Save these credentials to AWS SSM at the [paths specified by the `eks/argocd` component](/layers/software-delivery/eks-argocd/setup/#-configure-argo-cd-to-use-github-apps)
+          <Steps>
+           1. On the GitHub App page, scroll down to "Private keys" and click "Generate a private key"
+           1. Download the private key file
+           1. Store the App ID, Installation ID, and private key securely in 1Password
+          </Steps>
+      2. Upload these credentials to AWS SSM Parameter Store
+          <TaskList>
+          - Upload the `Argo CD Instance` private key to `/argocd/argo_cd_instance/app_private_key` `core-auto`: 
+            ```bash
+            # Replace acme with your namespace or assume the role separately.
+            # Your default region should be the same as your primary region. 
+            assume-role acme-core-gbl-auto-admin
+            chamber write argocd argo_cd_instance/app_private_key \
+              "$(cat /path/to/argocd-deploy-non-prod.private-key.pem)"
```

Comment:
I know chamber is pretty standard, but maybe we should use native aws cli to avoid another installed binary. This won't be a problem when atmos auth exists.